css/flag-icon.css

```diff
@@ -276,6 +276,12 @@
 .flag-icon-ch.flag-icon-squared {
   background-image: url(../flags/1x1/ch.svg);
 }
+.flag-icon-cht {
```

Comment:
https://en.wikipedia.org/wiki/Chinese_Taipei_at_the_Olympics The IOC code is TPE, you should probably use TPE.